waspc/examples/todoApp/src/server/queries.ts

```diff
@@ -23,19 +23,19 @@ export const getTasks: GetTasks<Task[]> = async (args, context) => {
   return tasks
 }
 
-export const getNumTasks: GetNumTasks<number> = async (args, context) => {
+export const getNumTasks: GetNumTasks = async (_args, context) => {
   return context.entities.Task.count()
 }
 
-export const getTask: GetTask<Task> = async ({ id }, context) => {
+export const getTask: GetTask<Pick<Task, 'id'>> = async (where, context) => {
```

Comment:
I'd argue that changing this one from `{ id }` to `where` communicates less information than before, but I guess it could make sense now that we have the type. Your call!

Reply:
I agree, but I like how passing the argument wholesale illustrates that quite often `args` is indeed equivalent to a `where` clause, and it just slots in so neatly, now that it is typed. Still a tough call, but since you're leaving it up to me, I'll keep it like this.

Reply:
Sure! It's our internal testing app anyway. We usually go crazy with stuff here, so naming is the least of our concerns.